.kitchen.docker.yml

```diff
@@ -90,9 +89,9 @@ platforms:
     intermediate_instructions:
       - RUN zypper --non-interactive install aaa_base perl-Getopt-Long-Descriptive which net-tools
 
-- name: opensuse-42.1
+- name: opensuse-42.2
```

Comment:
We can set this to -leap now which will probably save time updating it everytime they ship a release